### PR TITLE
Collection of create form UX improvements

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

Includes various UX improvements to the create form, as well as some cosmetic bug fixes.

- reduce placeholder text
- make preview more subtle
- fix a few static loading states in preview
- disable pay input in preview

## Screenshots or screen recordings

| before | after |
| --- | --- |
| Screen Shot 2022-06-20 at 9 52 20 PM | Screen Shot 2022-06-20 at 10 25 39 PM |

## Acceptance checklist

- [ ] I have evaluated the
      [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines)
      for this PR.
- [ ] I have tested this PR in
      [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/shared/formItems/ProjectLink.tsx [hunk 1]
```diff
@@ -14,11 +14,7 @@ export default function ProjectLink({
       label={hideLabel ? undefined : t`Website`}
       {...formItemProps}
     >
-      <Input
-        placeholder={t`https://your-project.com`}
-        type="url"
```
Comment (aeolianeth) [inline src/components/shared/formItems/ProjectLink.tsx:19]: remove type=url for better UX

File: src/components/shared/inputs/Pay/PayInputGroup.tsx [hunk 2]
```diff
@@ -81,11 +81,13 @@ export default function PayInputGroup({
             }}
             value={payAmount}
             min={0}
+            disabled={disabled}
```
Comment (aeolianeth) [inline src/components/shared/inputs/Pay/PayInputGroup.tsx:84]: disable the input and the accessory

File: src/components/v2/V2Create/ProjectConfigurationFieldsContainer.tsx [hunk 3]
```diff
@@ -9,41 +9,35 @@ const FULL_WIDTH_PX = 24
 
 export default function ProjectConfigurationFieldsContainer({
   showPreview,
-  previewContent,
```
Comment (aeolianeth) [inline src/components/v2/V2Create/ProjectConfigurationFieldsContainer.tsx:12]: remove unused prop

File: src/components/v2/V2Project/TreasuryStats/index.tsx [hunk 4]
```diff
@@ -28,11 +29,12 @@ export default function TreasuryStats() {
 
   const { data: overflow, loading: overflowLoading } =
     useTerminalCurrentOverflow({
-      terminal: terminals?.[0],
+      terminal: primaryTerminal,
```
Comment (aeolianeth) [inline src/components/v2/V2Project/TreasuryStats/index.tsx:32]: just a smol improvement while passing by

File: src/components/v2/V2Project/TreasuryStats/index.tsx [hunk 5]
```diff
@@ -28,11 +29,12 @@ export default function TreasuryStats() {
...
       projectId,
     })
 
-  const fundingProgressBarLoading = overflowLoading || distributionLimitLoading
+  const fundingProgressBarLoading =
```
Comment (aeolianeth) [inline src/components/v2/V2Project/TreasuryStats/index.tsx:36]: fix preview mode progress bar

File: src/components/v2/V2Project/V2FundingCycleSection/ReservedTokensSplitsCard.tsx [hunk 6]
```diff
@@ -88,7 +88,7 @@ export default function ReservedTokensSplitsCard({
             <div style={{ marginRight: '3rem' }}>
               <Skeleton
                 active
-                loading={loadingReservedTokens}
+                loading={!isPreviewMode && loadingReservedTokens}
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/ReservedTokensSplitsCard.tsx:91]: fix preview mode reserve tokens

File: src/styles/antd-overrides/input.scss [hunk 7]
```diff
@@ -300,3 +300,11 @@ textarea.ant-input::placeholder {
 .ant-form-item-label > label .ant-form-item-optional {
   color: var(--text-primary);
 }
+
+.ant-input-number-disabled {
+  border: 1px solid var(--stroke-tertiary);
```
Comment (aeolianeth) [inline src/styles/antd-overrides/input.scss:305]: add an input disabled state.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -1,18 +1,13 @@
...
+_If applicable, provide screenshots or screen recordings to demonstrate the changes._
 
 ## Acceptance checklist
 
-- [ ] I have evaluated the
-      [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines)
-      for this PR.
-- [ ] I have tested this PR in
-      [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
+- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
```
Comment (aeolianeth) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: fix annoying newline in checklist

File: .prettierrc [hunk 9]
```diff
@@ -6,5 +6,13 @@
   "singleQuote": true,
   "tabWidth": 2,
   "semi": false,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "overrides": [
+    {
+      "files": "*.md",
+      "options": {
+        "printWidth": 600
```
Comment (aeolianeth) [inline .prettierrc:14]: make line length for md files longer (no need to be 80 chars)

File: src/components/shared/Project/ProjectHeader/index.tsx [hunk 10]
```diff
@@ -25,6 +25,8 @@ export default function ProjectHeader({
   const headerHeight = 120
   const spacing = 20
 
+  const projectTitle = metadata?.name || t`Untitled project`
```
Comment (benschac) [inline src/components/shared/Project/ProjectHeader/index.tsx:28]: super tiny nit: for nullish checks does it make more sense to use `??`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**